hack/gencerts.sh

```diff
@@ -127,7 +131,22 @@ if [ "$IN_POD" == "true" ];  then
 	    "server-key.pem": "'"$server_key"'"
 	  }
 	}' \
-	https://kubernetes.default.svc/api/v1/namespaces/${NAMESPACE}/secrets
+	https://kubernetes.default.svc/api/v1/namespaces/${NAMESPACE}/secrets)
+
+	cat ${TMP_DIR}/output
+
+	case "$STATUS" in
+	  201)
+	    printf "\nSuccess - secret created.\n"
+	  ;;
+	  409)
+	    printf "\nSuccess - secret already exists.\n"
+	  ;;
+	  *)
+	    printf "\nFailed creating secret.\n"
+	    exit 1
```

Comment:
@liyinan926 @yuchaoran2011 now I explicitly fail on any error like 
```
Creating a secret for the certificate and keys
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  4  8033  100   329    0     0   1920      0 --:--:-- --:--:-- --:--:--  1912
HTTP/1.1 403 Forbidden
Content-Type: application/json
X-Content-Type-Options: nosniff
Date: Fri, 21 Sep 2018 13:28:41 GMT
Content-Length: 329
Connection: close

{
  "kind": "Status",
  "apiVersion": "v1",
  "metadata": {

  },
  "status": "Failure",
  "message": "secrets is forbidden: User \"system:serviceaccount:sparkoperator:sparkoperator\" cannot create secrets in the namespace \"sparkoperator\"",
  "reason": "Forbidden",
  "details": {
    "kind": "secrets"
  },
  "code": 403
}
Failed creating secret.
```

and succeed with 
```
HTTP/1.1 409 Conflict
Content-Type: application/json
Date: Fri, 21 Sep 2018 13:31:08 GMT
Content-Length: 277

{
  "kind": "Status",
  "apiVersion": "v1",
  "metadata": {

  },
  "status": "Failure",
  "message": "secrets \"spark-webhook-certs\" already exists",
  "reason": "AlreadyExists",
  "details": {
    "name": "spark-webhook-certs",
    "kind": "secrets"
  },
  "code": 409
}
Success - secret already exists.
```

or 

```
HTTP/1.1 201 Created
Content-Type: application/json
Date: Fri, 21 Sep 2018 13:32:32 GMT
Transfer-Encoding: chunked

{
  "kind": "Secret",
  "apiVersion": "v1",
  "metadata": {
    "name": "spark-webhook-certs",
    "namespace": "sparkoperator",
    "selfLink": "/api/v1/namespaces/sparkoperator/secrets/spark-webhook-certs",
    "uid": "redacted",
    "resourceVersion": "2384255",
    "creationTimestamp": "2018-09-21T13:32:32Z"
  },
  "data": {
    "ca-cert.pem": "redacted",
    "ca-key.pem": "redacted",
    "server-cert.pem": "redacted",
    "server-key.pem": "redacted"
  },
  "type": "Opaque"
}
Success - secret created.
```